### DISCUSSION TITLE
Unable to connect wit Lumia Stream

### DISCUSSION THREAD

**Discussion by BluesBroBo** [pear-devs/pear-desktop#1973] [Q&A] [answered]
I wanted to use the Youtube Music Desktop App with lumia stream to have my viewers choose music and get info on what is currently playing. I enabled the Lumia Addon in the app and also tried to setup the connector in Lumia, but I can't get it to work. Lumia wants a port any possibly a password for this, for both I don't know what is the right one :/
I can choose between another Youtube Music App which is offline by now and this one in the connector, but after I save the setting it just automatically switch back to the other one.
I have no real clue what I have to do to get this to work. Can someone help?

**Comment by ArjixWasTaken** [comment]
https://github.com/th-ch/youtube-music/issues/1893

**Reply by BluesBroBo** [reply]
There is a lumia addon within this app, not the ymtdesktop version but this one, I did not mix up those two.

The plugin is listed on the main page of this repository (https://github.com/th-ch/youtube-music?tab=readme-ov-file#available-plugins) and can be activated within the plugins of the app.
![image](https://github.com/th-ch/youtube-music/assets/77116547/4353e432-72a1-4ba2-bf88-f8c36b738660)

I imagined it would work with the already existing connector within the lumia app, but that doesn't seem to be the case. Therefore I wanted to ask how I can connect this app with lumia, since there clearly is the implication that it is supposed to be doable.

**Reply by ArjixWasTaken** [reply]
you are correct, sorry for my rushed response

**Reply by ArjixWasTaken** [reply]
the lumia plugin sends data to `127.0.0.1:39231`, is lumia listening for messages on that port?

**Reply by ArjixWasTaken** [reply, marked as answer]
made an official issue here

https://github.com/th-ch/youtube-music/issues/2003

**Reply by BluesBroBo** [reply]
Thank you very much! ♥️
